### DISCUSSION TITLE
Themebuilder

### DISCUSSION THREAD

**Discussion by Abooki2023** [elementor/elementor#27934] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I cant edit some pages

### Describe the solution you'd like.

Allowing emulator to edit some of my pages

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by arsanet** [comment]
Hey! which pages you want to edit that current Elementor Theme Builder can't help you?
